scripts/eslint/rules/utils.js

```diff
@@ -0,0 +1,2 @@
+const { ESLintUtils } = require("@typescript-eslint/utils");

```

Comment:
TODO: remove this; ESLint's createRule works exactly like this; we don't have to duplicate it.

Reply:
No, it doesn't, nevermind.